web-build/Dockerfile

```
@@ -1,5 +1,4 @@
 #ddev-generated
+
 # Note that the chromium-driver install could be moved to webimage_extra_packages
 RUN sudo apt-get update && sudo apt-get install chromium-driver -y
```

```suggestion
RUN sudo apt-get update && sudo apt-get install chromium-driver -y

ARG TARGETPLATFORM
RUN SQLITE_VERSION=3.45.1 && \
    mkdir -p /tmp/sqlite3 && \
    wget -O /tmp/sqlite3/sqlite3.deb [https://ftp.debian.org/debian/pool/main/s/sqlite3/sqlite3_${SQLITE_VERSION}-1_${TARGETPLATFORM##linux/}.deb](https://ftp.debian.org/debian/pool/main/s/sqlite3/sqlite3_$%7BSQLITE_VERSION%7D-1_$%7BTARGETPLATFORM##linux/}.deb) && \
    wget -O /tmp/sqlite3/libsqlite3.deb [https://ftp.debian.org/debian/pool/main/s/sqlite3/libsqlite3-0_${SQLITE_VERSION}-1_${TARGETPLATFORM##linux/}.deb](https://ftp.debian.org/debian/pool/main/s/sqlite3/libsqlite3-0_$%7BSQLITE_VERSION%7D-1_$%7BTARGETPLATFORM##linux/}.deb) && \
    dpkg -i /tmp/sqlite3/*.deb && \
    rm -rf /tmp/sqlite3
```
the snippet for upgrading SQLIte to 3.45.1 suggested by @stasadev on https://github.com/ddev/ddev/issues/6110